zest/releaser/tests/gitsvn.txt

```diff
@@ -80,7 +80,7 @@ Originally there are no tags:
 
 Create a tag and it will show up (master diverges from trunk):
 
-    >>> cmds = checkout.cmd_create_tag('0.1')
+    >>> cmds = checkout.cmd_create_tag('0.1', 'Create tag 0.1')
```

Comment:
Is it necessary to use the non-default (and git style) "create tag" message here instead of the default "Tagging" one? It confused me initially.

Reply:
It's not strictly necessary, but this would detect if `cmd_create_tag()` would not use the message from the argument but still the old default.

Reply:
Good point!

Could you change "create" to "creating"? I totally utterly detest git's imperative style :-) (I even helped to keep Django from adopting this style :-) )